gradio/blocks.py

```diff
@@ -514,6 +515,7 @@ def __init__(
             css: Custom css as a string or path to a css file. This css will be included in the demo webpage.
             js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
             head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
+            expand_to_height: Whether to vertically expand top-level child components to the height of the window. If True, expansion occurs when the scale value of the child components >= 1.
```

Comment:
This means that the Gradio app will be the full height of the window, right? I think `full_height` would be a better parameter name

Reply:
Should we add this parameter to `Interface` and `TabbedInterface` too btw?

Reply:
So just to understand, the same `scale` parameter is used to determine how much the height of each child component should expand by? If so, we should update the docstring for `scale` for each component to mention this. If not, how does this work if you have multiple child elements whose height can vary?

Reply:
>  I think full_height or fill_height would be a better parameter name

Renamed to `fill_height`. 

> Should we add this parameter to Interface and TabbedInterface too btw?

I don't think it's necessary. 

> If so, we should update the docstring for scale for each component to mention this.

Ok updated docstring of every component.